src/css/profile/wearable/changeable/common/button.less

```diff
@@ -355,3 +355,10 @@ button.ui-btn, input.ui-btn {
 		margin-top: 0;
 	}
 }
+
+.ui-button-go-to-top {
+	width: 52 * @unit_base;
+	height: 52 * @unit_base;
+	background-image: url("./images/GoToTopButton/go_to_top_dark.png");
```

Comment:
I think we shouldn't create additional directory like 'GoToTopButton' to store images. This makes additional issues when images will be updated.

Reply:
We are having mages in dirs https://github.com/Samsung/TAU/tree/tau_1.0/src/css/profile/wearable/changeable/theme-changeable/images